test/e2e/resources/components/componentGlobalUsage.brs

```diff
@@ -0,0 +1,7 @@
+sub main()
+    ' create an object that uses m.global
+    myComponent = createObject("roSGNode", "MGlobalWidget")
+    mySecondComponent = createObject("roSGNode", "MGlobalWidget")
```

Comment:
Could this be another type of component so it's clear that works among components?